templates/request_page.hbs

```diff
@@ -75,11 +75,9 @@
         </div>
 
         <div class="comment-container">
-          <div class="comment-show-container {{#validate 'body'}}hidden{{/validate}}">
-            <span class="comment-show-container-content">
-              {{t 'add_to_conversation'}}
-            </span>
-          </div>
+          <button type="button" class="comment-show-container {{#validate 'body'}}hidden{{/validate}}">
```

Comment:
just noticed, why `type="button"` if it's already a `<button />`?

Reply:
Good question! Because by default type will be set to `submit` which will.. well, submit the form and therefore reload the page.

Reply:
Ahhh... interesting!